src/sass/modules/_m-schemaform.scss

```diff
@@ -231,3 +231,41 @@ label {
     padding-left: 25px !important;
   }
 }
+
+//duplicated from disability-benefits.scss
```

Comment:
Should these be rename and reused by relocating in base/_va.scss? Or should another ticket be created for that?